src/main/java/com/beis/subsidy/award/transperancy/dbpublishingservice/model/Beneficiary.java

```diff
@@ -0,0 +1,101 @@
+package com.beis.subsidy.award.transperancy.dbpublishingservice.model;
+
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+/**
+ * 
+ * Beneficiary Entity class
+ *
+ */
+@Builder
+@Entity(name = "BENEFICIARY")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Beneficiary {
+
+	
+	@Id 
+	@GeneratedValue(strategy = GenerationType.AUTO,generator="beneficiary_read_seq")
+	@SequenceGenerator(name = "beneficiary_read_seq",sequenceName="beneficiary_read_seq",allocationSize=1)
```

Comment:
Auto generator not required here since already using sequence generator . please remove from here.